app/src/features/skjema-moduler/Inntekt.tsx

```diff
@@ -272,9 +284,12 @@ export const ENDRINGSÅRSAK_TEMPLATE = {
   årsak: "" as const,
 };
 
-function Endringsårsaker() {
-  const { eksisterendeInntektsmeldinger } = useLoaderData({ from: "/$id" });
```

Comment:
Litt irriterende å måtte sende inn dette fra toppen, men kom ikke på noen bedre løsning. Åpen for innspill!

Reply:
Enig. Er at av problemene jeg ikke føler jeg har "løst" med RouteLoaders. Med engang det er noe som er felles på tvers av routes går man tilbake til container/component pattern, som føles feil.

Er ev. å lage en context som man populerer på Route nivå. Men det føles dobbelt opp, da Route allerede er en context under panseret.

Hadde vært nice hvis man kunne sagt `useLoaderData` uten en path og akseptert at den tryner om man er på feil path på samme måte som med contextprovider generelt 🤔

Reply:
Det tryner om man er på feil path faktisk, men du må jo spesifisere det da. Og fint å få typesikkerheten ved å spesifisere da.

Men men, det ble jo ikke SÅ grisete å sende ned heller.